lib/generated/intl/messages_en.dart

```diff
@@ -19,6 +19,402 @@ typedef String MessageIfAbsent(String messageStr, List<dynamic> args);
+        "marketStats": MessageLookupByLibrary.simpleMessage("market stats"),
+        "marketcap": MessageLookupByLibrary.simpleMessage("Marketcap"),
+        "markets": MessageLookupByLibrary.simpleMessage("Markets"),
+        "maxSupply": MessageLookupByLibrary.simpleMessage("Max Supply"),
+        "mnemonicPhrase":
+            MessageLookupByLibrary.simpleMessage("Mnemonic Phrase"),
+        "month1": MessageLookupByLibrary.simpleMessage("1 month"),
+        "moreDetails": MessageLookupByLibrary.simpleMessage("More Details"),
+        "myAccount": MessageLookupByLibrary.simpleMessage("My Account"),
+        "network": MessageLookupByLibrary.simpleMessage("Network"),
+        "next": MessageLookupByLibrary.simpleMessage("Next"),
+        "notificationDetails":
+            MessageLookupByLibrary.simpleMessage("Notifications Details"),
+        "notificationSettings":
+            MessageLookupByLibrary.simpleMessage("Notification Settings"),
+        "notifications": MessageLookupByLibrary.simpleMessage("Notifications"),
+        "notificationsCenter":
+            MessageLookupByLibrary.simpleMessage("Notifications Center"),
+        "notifyWhenAFunds": MessageLookupByLibrary.simpleMessage(
+            "Notify when a funds are received."),
+        "notifyWhenAWithdrawal": MessageLookupByLibrary.simpleMessage(
+            "Notify when a withdrawal is requested."),
+        "notifyWhenAnOrder": MessageLookupByLibrary.simpleMessage(
+            "Notify when an order is filled."),
+        "oneOrMoreOfYour1224Words": MessageLookupByLibrary.simpleMessage(
+            "One or more of your 12-24 words are incorrect, make sure that the order is correct or if there is a typing error."),
+        "online": MessageLookupByLibrary.simpleMessage("Online"),
+        "openOrders": MessageLookupByLibrary.simpleMessage("Open Orders"),
+        "optimize": MessageLookupByLibrary.simpleMessage("Optimize"),
+        "orderBook": MessageLookupByLibrary.simpleMessage("Order Book"),
+        "orderTypes": MessageLookupByLibrary.simpleMessage("Order Types"),
+        "ordersHistory": MessageLookupByLibrary.simpleMessage("Orders History"),
+        "others": MessageLookupByLibrary.simpleMessage("Others"),
+        "pair": MessageLookupByLibrary.simpleMessage("Pair"),
+        "participateInLuckyDraw": MessageLookupByLibrary.simpleMessage(
+            "Participate in lucky draw and share 160 USDT prize poll!"),
+        "password": MessageLookupByLibrary.simpleMessage("Password"),
+        "phone": MessageLookupByLibrary.simpleMessage("Phone"),
+        "pleaseEnterAValidPrice":
+            MessageLookupByLibrary.simpleMessage("Please enter a valid price"),
+        "pleaseEnterAgain":
+            MessageLookupByLibrary.simpleMessage("Please enter again."),
+        "pleaseEnterThe1224Words": MessageLookupByLibrary.simpleMessage(
+            "Please enter the 12-24 words in the correct order."),
```

Comment:
```suggestion
            "Please rearrange the 12-24 words in the correct order."),
```

This user clicked on the words in the correct order and nothing happened